### ISSUE TITLE
iPhone X 14.5

### ISSUE THREAD

**Issue by waqasnasir4** [checkra1n/BugTracker#2081]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone X
2. On what version of iOS is it?
iOS 14.5
3. What version of checkra1n are you using?
0.12.3 Beta
4. What is your host system (OS version? Hackintosh? VM? etc.)?
MacBook Air Big Sur
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB C to hub to USB A to iPhone 
**What are the steps to reproduce the issue?**
1. No password or Face ID 
2.Start checkrain, open options click bypass A11
3. Press start checkrain puts iPhone into recovery mode, follow directions for dfu and press start
4. Before completion get error message
“Timed out waiting for bootstrap upload (likely caused by incompatible tweak)
I also select options for checkrain safe mode 
Same error but when phone boots up it says checkrain booted into safe mode per user request but still no checkrain icon
...

**What do you expect, and what is happening instead?**
Phone to be jailbroken but it’s not working 

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**

yes

**Any other info, error logs, screenshots, ...?**
I have screenshots for error screen 
https://ibb.co/5Tz1mDh

**Comment by Daniel0829** [comment]
I am in the same situation, exactly according to the specification, but there is an error -20, I do not know where the problem is. I checked safe mode, but the problem is still recurring.
I have repeated the test about 10 times, but it did not work, each time the error -20 appears!

**Comment by Daniel0829** [comment]
What iDevice are you using?
iPhone X
On what version of iOS is it?
iOS 14.5
What version of checkra1n are you using?
0.12.3 Beta
What is your host system (OS version? Hackintosh? VM? etc.)?
MacBook Pro Catalina 10.15.7
How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB C to hub to USB A to iPhone

**Comment by waqasnasir4** [comment]
Hey I solved it Daniel and for everyone that is having the same problem as me. Right click on Checkrain in finder and click show content. Open the terminal version of Checkrain and select all options and follow the steps as you normally would and that should solve your problem hopefully.

**Comment by Daniel0829** [comment]
Thank you waqasnasir4. I followed your method to re-jailbreak and it worked. Thank you very much for the method!

**Comment by hasimy-as** [comment]
Thanks @waqasnasir4, your method work but after it's all done I don't see the checkrain icon anywhere on my phone. Any suggestions?

**Comment by waqasnasir4** [comment]
> Thanks @waqasnasir4, your method work but after it's all done I don't see the checkrain icon anywhere on my phone. Any suggestions?

After it’s all done, just wait a few minutes and it should pop up.

**Comment by waqasnasir4** [comment]
> Thanks @waqasnasir4, your method work but after it's all done I don't see the checkrain icon anywhere on my phone. Any suggestions?

If that doesn’t work 
Follow this guide
https://github.com/checkra1n/BugTracker/issues/2077

**Comment by hasimy-as** [comment]
Yeah, it's been a few minutes and it's still not showing up. I will try that guide though, thanks!

**Comment by fdofdez** [comment]
> Hey I solved it Daniel and for everyone that is having the same problem as me. Right click on Checkrain in finder and click show content. Open the terminal version of Checkrain and select all options and follow the steps as you normally would and that should solve your problem hopefully.

To have the checkra1n app, after the JB process (save mode selected), the iphone will boot normally saying it is in save mode, now run again checkra1n without save mode, and there you will have the app. 

After the boot you will be in JB State. If you backed up your cydia info, after the restoration you have the JB again firts on saved mode and later without save mode.

**Comment by QziP22** [comment]
> 
> 
> Hey I solved it Daniel and for everyone that is having the same problem as me. Right click on Checkrain in finder and click show content. Open the terminal version of Checkrain and select all options and follow the steps as you normally would and that should solve your problem hopefully.

i tried this method and didnt work. it still gets stuck on the booting part, white screen with the checkra1n logo, then i get the -20 error.
ipad pro 9.7 inch
MacbookPro-BigSur 11.3.1, 0.12.3 checkra1n
Usb c to a adapter
